lib/Jsrt/ChakraCore.h

```diff
@@ -292,7 +292,13 @@ CHAKRA_API
 /// <param name="value">JavascriptString value</param>
 /// <param name="buffer">Pointer to buffer</param>
 /// <param name="bufferSize">Buffer size</param>
-/// <param name="written">Total number of characters written</param>
+/// <param name="writtenLength">Total number of characters written. This is only
```

Comment:
I think this should also be clarified in the same way as the `actualLength` parameter.

Might also be worth clarifying that there is no trailing null character?